frontend/src/features/permits/hooks/hooks.ts

```diff
@@ -382,16 +382,20 @@ export const useApplicationsInProgressQuery = ({
   page = 0,
   take = 10,
   searchString = "",
-  sorting = [],
+  orderBy = [],
+}: {
+  page?: number;
+  take?: number;
+  searchString?: string;
+  orderBy?: SortingConfig[];
 }) => {
-  const applicationsInProgressQuery = useQuery({
-    queryKey: ["applicationInProgress"],
+  return useQuery({
+    queryKey: ["applicationsInProgress", page, take],
```

Comment:
Put back the orderBy as part of the queryKey as well, refer to the original queryKey that was removed